ckan/templates-bs2/package/base.html

```diff
@@ -16,7 +16,7 @@
 	{% else %}
 	    <li>{% link_for _('Datasets'), named_route='dataset.search' %}</li>
 	{% endif %}
-	<li{{ self.breadcrumb_content_selected() }}>{% link_for dataset|truncate(30), named_route='dataset.read', id=pkg.name %}</li>
+	<li{{ self.breadcrumb_content_selected() }}>{% link_for dataset|truncate(30), named_route='dataset.read', id=pkg.name if is_activity_archive else pkg.name %}</li>
```

Comment:
the if/else looks like a no-op. Did you mean to use the id instead of the name when is_activity_archive is true?

Reply:
good spot! changed